020_up_L22.py

```
@@ -0,0 +1,128 @@
+    args = parser.parse_args()
+
+    load_dotenv(args.config)
+    tmpdir = path.dirname(path.abspath(__file__)) + '/tmp'
+    command = genslice(path.abspath(args.file), args.time)
+
+    if sameparams(tmpdir, command):
+        os.chdir(tmpdir)
+    else:
+        os.mkdir(tmpdir)
+        os.chdir(tmpdir)
+        os.system(command)
```

Ruff (0.14.5)

Starting a process with a shell, possible injection detected

(S605)